src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -90,6 +90,8 @@ export default {
 			unregisterTab: this.unregisterTab,
 			// Getter as an alternative to Vue 2.7 computed(() => this.activeTab)
 			getActiveTab: () => this.activeTab,
+			// Used to check whether the tab header is shown so the tabs can reference the tab header for `aria-labelledby` or not
+			tablistShown: () => this.hasMultipleTabs,
```

Comment:
If we provide a getter function, I'd explicitly call it as a function, as `getActiveTab` above

```suggestion
			// Used to check whether the tab header is shown so the tabs can reference the tab header for `aria-labelledby` or not
			getTablistShown: () => this.hasMultipleTabs,
```

Reply:
:+1: